pkg/server/serverpb/admin.proto

```diff
@@ -227,6 +227,8 @@ message NonTableStatsRequest {
 message NonTableStatsResponse {
   // Information on time series ranges,
   TableStatsResponse time_series_stats = 1;
+  // Information for remaining non-table [non-time-series] ranges
```

Comment:
or maybe "non-table, non-time-series"

Reply:
or maybe, since it's further clarifying remaining, "remaining (non-table, non-time-series) ranges"?

...maybe spending too much time thinking about the punctuation of a comment...